aspnetcore/signalr/hubs.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Use hubs in ASP.NET Core SignalR
+author: rachelappel
+description: Learn how to use hubs in ASP.NET Core SignalR.
+manager: wpickett
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 03/30/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/hubs
+---
+
+# Use hubs in SignalR for ASP.NET Core
+
+By [Rachel Appel](https://twitter.com/rachelappel) and [Kevin Griffin](https://twitter.com/1kevgriff)
+
+## What is a SignalR hub
+
+The SignalR Hubs API enables you to call methods on connected clients from the server. In the server code, you define methods that are called by client. In the client code, you define methods that are called from the server. SignalR takes care of everything behind the scenes that makes real-time client-to-server and server-to-client communications possible.
+
+## Configure SignalR hubs
+
+The SignalR middleware requires some services to be able to run. To configure those call `services.AddSignalR`.
+
+[!code-javascript[Configure service](hubs/sample/startup.cs?range=35)]
+
+When adding SignalR functionality to an ASP.NET Core app, setup SignalR routes by calling `app.UseSignalR` in the `Startup.Configure` method.
+
+[!code-javascript[Configure routes to hubs](hubs/sample/startup.cs?range=55-58)]
+
+## Create and use hubs
+
+Create a hub by declaring a class that inherits from `Hub`, and add public methods to it. Clients can call methods that are defined as `public`.
+
+[!code-csharp[Create and use hubs](hubs/sample/hubs/chathub.cs?range=10-14)]
+
+You can specify a return type and parameters, including complex types and arrays, as you would in any C# method. SignalR handles the serialization and deserialization of complex objects and arrays in your parameters and return values.
+
+## The Clients object
+
+Each instance of the `Hub` class has a property named `Clients` that contains the following members for communication between server and client:
+
+| Property | Description |
+| ------ | ----------- |
+| `All` | Calls a method on all connected clients |
+| `Caller` | Calls a method on the currently connected client |
+| `Others` | Calls a method on all connected clients except the client that invoked the method |
+
+Additionally, the `Hub` class contains the following methods:
+
+| Method | Description |
+| ------ | ----------- |
+| `AllExcept` | Calls a method on all connected clients except for the provided connections |
+| `Client` | Calls a method on a specific connected client |
+| `Clients` | Calls a method on specific connected clients |
+| `Group` | Sends a message to all connections in the specified group  |
+| `GroupExcept` | Sends a message to all connections in the specified group, except the specified connections |
+| `Groups` | Sends a message to multiple groups of connections  |
+| `OthersInGroup` | Sends a message to others in a group of connections  |
```

Comment:
"others" needs definition. How about "Sends a message to a group of connections, excluding the client that invoked the Hub method"